### PR TITLE
build for wetc

### PR DESCRIPTION
**Pull request by l303l**

### CODE REVIEW

**Review by gitr0n1n — CHANGES_REQUESTED** [review]
networks.json
Ethereum Classic Networks:
ETC Mainnet = "61"
KOT Kotti = "6"
METC Mordor = "63" 
TETC Morden = "62"

Source: https://chainlist.org/

**Comment by gitr0n1n** [comment]
> networks.json Ethereum Classic Networks: ETC Mainnet = "61" KOT Kotti = "6" METC Mordor = "63" TETC Morden = "62"
> 
> Source: https://chainlist.org/

Looks like Morden was replaced by Mordor. So only three active networks for ETC.
Source: http://ethereumclassic.org/development/testnets

**Review by gitr0n1n — APPROVED** [review]
LGTM

File: build/contracts/WETC9.json [hunk 1]
```diff
@@ -279,11 +279,11 @@
...
...59;;;;;2276:2;2241:9;:14;2251:3;2241:14;;;;;;;;;;;;;;;:26;2256:10;2241:26;;;;;;;;;;;;;;;;:38;;2220:59;2216:179;;;2333:3;2303:9;:14;2313:3;2303:14;;;;;;;;;;;;;;;:26;2318:10;2303:26;;;;;;;;;;;;;;;;:33;;2295:42;;;;;;;;2381:3;2351:9;:14;2361:3;2351:14;;;;;;;;;;;;;;;:26;2366:10;2351:26;;;;;;;;;;;;;;;;:33;;;;;;;;;;;2216:179;2423:3;2405:9;:14;2415:3;2405:14;;;;;;;;;;;;;;;;:21;;;;;;;;;;;2454:3;2436:9;:14;2446:3;2436:14;;;;;;;;;;;;;;;;:21;;;;;;;;;;;2487:3;2473:23;;2482:3;2473:23;;;2492:3;2473:23;;;;;;;;;;;;;;;;;;2514:4;2507:11;;2065:460;;;;;:::o;1445:203::-;1523:3;1498:9;:21;1508:10;1498:21;;;;;;;;;;;;;;;;:28;;1490:37;;;;;;;;1562:3;1537:9;:21;1547:10;1537:21;;;;;;;;;;;;;;;;:28;;;;;;;;;;;1575:10;:19;;:24;1595:3;1575:24;;;;;;;;;;;;;;;;;;;;;;;;8:9:-1;5:2;;;45:16;42:1;39;24:38;77:16;74:1;67:27;5:2;1575:24:1;1625:10;1614:27;;;1637:3;1614:27;;;;;;;;;;;;;;;;;;1445:203;:::o;822:27::-;;;;;;;;;;;;;:::o;1108:65::-;;;;;;;;;;;;;;;;;:::o;785:31::-;;;;;;;;;;;;;;;;;;;;;;;;;;;;;;;;;;;;;;;;;;;;;;;;;;;;;;;;;;;;;;;;;;;;;;;;;;;;;;;;;;;;;;;;;;;;;;;;;;;;;;;;;;;;;;;;;;;;;;;;;;:::o;1938:121::-;1995:4;2018:34;2031:10;2043:3;2048;2018:12;:34::i;:::-;2011:41;;1938:121;;;;:::o;1179:65::-;;;;;;;;;;;;;;;;;;;;;;;;;;:::o",
```
Comment (ETCSecurity) [inline build/contracts/WETC9.json:285]: Does this need to be changed @l303l ?

Comment (ETCSecurity) [reply build/contracts/WETC9.json:285]: Point has been clarified. Resolving issue.

**Review by ETCSecurity — APPROVED** [review]

**Pull request merged**